test/src/api/responder04Test.php

```diff
@@ -0,0 +1,916 @@
+            $retval, array(array(0, 2), array(1, 3)),
+            $gameId, 2, 'Skill', 0, 1, '', array(2 => 11));
+
+        $expData['activePlayerIdx'] = 1;
+        array_pop($expData['gameActionLog']);
+        array_pop($expData['gameActionLog']);
+        array_unshift($expData['gameActionLog'], array('timestamp' => 'TIMESTAMP', 'player' => 'responder003', 'message' => 'responder003 performed Skill attack using [p(V=10)!:6] against [D(12):6]; Defender D(12) was captured; Attacker p(V=10)! rerolled from 6'));
+        array_unshift($expData['gameActionLog'], array('timestamp' => 'TIMESTAMP', 'player' => 'responder003', 'message' => 'Turbo die p(V=10)! changed size from 10 to 11 sides, recipe changed from p(V=10)! to p(V=11)!, rolled 4'));
+        $expData['gameActionLogCount'] = 24;
+        $expData['playerDataArray'][0]['activeDieArray'][2]['description'] = "Poison Turbo V Swing Die (with 11 sides)";
+        $expData['playerDataArray'][0]['activeDieArray'][2]['properties'] = array("HasJustTurboed");
+        $expData['playerDataArray'][0]['activeDieArray'][2]['sides'] = 11;
+        $expData['playerDataArray'][0]['activeDieArray'][2]['value'] = 4;
+        $expData['playerDataArray'][0]['capturedDieArray'][0]['properties'] = array("WasJustCaptured");
+        $expData['playerDataArray'][0]['capturedDieArray'][0]['recipe'] = "D(12)";
+        $expData['playerDataArray'][0]['capturedDieArray'][0]['sides'] = 12;
+        $expData['playerDataArray'][0]['capturedDieArray'][0]['value'] = 6;
+        $expData['playerDataArray'][0]['roundScore'] = 12;
+        $expData['playerDataArray'][0]['sideScore'] = -15;
+        $expData['playerDataArray'][0]['waitingOnAction'] = false;
+        $expData['playerDataArray'][1]['activeDieArray'][1]['properties'] = array();
+        $expData['playerDataArray'][1]['activeDieArray'][3]['description'] = "Doppelganger X Swing Die (with 7 sides)";
+        $expData['playerDataArray'][1]['activeDieArray'][3]['recipe'] = "D(X)";
+        $expData['playerDataArray'][1]['activeDieArray'][3]['sides'] = 7;
+        $expData['playerDataArray'][1]['activeDieArray'][3]['value'] = 1;
+        array_pop($expData['playerDataArray'][1]['activeDieArray']);
+        $expData['playerDataArray'][1]['capturedDieArray'][0]['properties'] = array();
+        $expData['playerDataArray'][1]['roundScore'] = 34.5;
+        $expData['playerDataArray'][1]['sideScore'] = 15;
+        $expData['playerDataArray'][1]['waitingOnAction'] = true;
+
+        $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
+
+        $_SESSION = $this->mock_test_user_login('responder004');
+        $this->verify_api_submitTurn(
+            array(),
+            'responder004 chose to perform a Power attack using [D(4):1] against [p(V=11)!:4]; responder004 must turn down fire dice to complete this attack. ',
+            $retval, array(array(1, 0), array(0, 2)),
+            $gameId, 2, 'Power', 1, 0, '', array());
```

Comment:
And here's the attack that goes wrong --- it's *exactly* the same as the attack i called out above --- doppelganger attacks turbo, doesn't send turbo values.  The only thing that's different is that this attack is fire-assisted by the `oF(C)` (which, incidentally, was itself created via doppelganger attack, in case that turns out to matter).

So turbo shouldn't be happening.